### PR TITLE
Remove products app

### PR DESCRIPTION
**Pull request by jmsmkn**
The products app has been extracted into its own application. All features on GC for this are removed. The blog posts on the frontpage are now only the highlighted posts.

### CODE REVIEW

File: app/grandchallenge/core/views.py [hunk 1]
```diff
@@ -102,40 +102,11 @@ def get_context_data(self, **kwargs):
...
-                title="Find Certified Solutions",
-                bullet_points=[
-                    "Filter to easily find solutions to your clinical questions",
-                    "Compare product specifications",
-                    "Verify CE and FDA certification",
-                ],
-                url=reverse("products:product-list"),
-                url_title="Products",
-                image="images/products.png",
-            ),
```
Comment (amickan) [inline app/grandchallenge/core/views.py:115]: How does the feature overview on the hompage look now that we have 5 rather than 6 highlights?

Comment (jmsmkn) [reply app/grandchallenge/core/views.py:115]: Unbalanced but we can add something back later

Screenshot 2024-02-26 at 15 36 09

File: app/config/urls/root.py [hunk 2]
```diff
@@ -135,7 +132,11 @@ def handler500(request):
     path("summernote/", include("django_summernote.urls")),
     path(
         "aiforradiology/",
-        include("grandchallenge.products.urls", namespace="products"),
+        RedirectPath.as_view(netloc="radiology.healthairegister.com"),
+    ),
+    path(
+        "aiforradiology/<path:path>",
```
[the remark this answers is not in the document]
Comment (jmsmkn) [reply app/config/urls/root.py:138]: The original app was not consistent about using trailing slashes.

**Comment by daangeijs** [comment]
I have all the redirects in place for the companies and products:
/product/airs-medical-swiftmr/ -> /products/airs-medical-swiftmr/
You can try it out here:
https://radiology.healthairegister.com/product/airs-medical-swiftmr/

File: app/grandchallenge/core/views.py (hunk 1 above)
Comment (jmsmkn) [reply app/grandchallenge/core/views.py:115]: I think it's best to leave it out as it is not longer a feature of GC. We used to have an alternating list which handled odd items better but that was removed in https://github.com/comic/grand-challenge.org/pull/2062.

**Comment by pkcakeout** [comment]
I am getting this error for the dev server now. I am having a hunch that this might be due to the removal of the products-app?

~~~~~
Exception while running run() in 'scripts.development_fixtures'
Traceback (most recent call last):
  File "/app/manage.py", line 21, in <module>
    main()
  File "/app/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django_extensions/management/email_notifications.py", line 65, in run_from_argv
    super().run_from_argv(argv)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django_extensions/management/email_notifications.py", line 77, in execute
    super().execute(*args, **options)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django_extensions/management/utils.py", line 62, in inner
    ret = func(self, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django_extensions/management/commands/runscript.py", line 281, in handle
    run_script(script_mod, *script_args)
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django_extensions/management/commands/runscript.py", line 159, in run_script
    exit_code = mod.run(*script_args)
                ^^^^^^^^^^^^^^^^^^^^^
  File "/app/scripts/development_fixtures.py", line 88, in run
    _set_user_permissions(users)
  File "/app/scripts/development_fixtures.py", line 192, in _set_user_permissions
    add_product_perm = Permission.objects.get(codename="add_product")
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/poetry/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 637, in get
    raise self.model.DoesNotExist(
django.contrib.auth.models.Permission.DoesNotExist: Permission matching query does not exist.
~~~~~

**Pull request merged**